src/System.Runtime.Serialization.Xml/tests/SerializationTypes.cs

```diff
@@ -1285,6 +1285,18 @@ public void Add(DateTime value)
         }
     }
 
+    public class DateTimeDataTypeTimeType
```

Comment:
It's hard to tell what this type is about, what about TypeWithDateTimePropertyAsXmlTime?